### DISCUSSION TITLE
The instantiation is excessively deep and possibly infinite

### DISCUSSION THREAD

**Discussion by paur94** [KevinVandy/material-react-table#722] [Q&A]
Hello, I have only one column, but this error is being displayed. What can I do?
![image](https://github.com/KevinVandy/material-react-table/assets/60259252/0dc52d23-acd2-42f0-a1f5-a5d564b1c005)

**Comment by KevinVandy** [comment]
What version of MRT? Sometimes nested types can cause this error. accessorFn can be used instead of accessorKey to get around this.
